client/coral-configure/containers/ConfigureStreamContainer.js

```diff
@@ -53,18 +53,20 @@ class ConfigureStreamContainer extends Component {
 
   handleChange (e) {
     if (e.target && e.target.id === 'qboxenable') {
-      this.props.asset.settings.questionBoxEnable = e.target.checked;
+      this.state.settings.questionBoxEnable = e.target.checked;
```

Comment:
Use `setState` to change the state. Never directly manipulate the state in React.